frame/ddc-metrics-offchain-worker/src/lib.rs

```diff
@@ -66,15 +56,18 @@ struct MetricInfo {
     appPubKey: String,
     #[serde(deserialize_with = "de_string_to_bytes")]
     partitionId: Vec<u8>,
-    bytes: u128,
-    requests: u128,
+    storageBytes: u128,
+    wcuUsed: u128,
+    rcuUsed: u128,
 }
 
+#[allow(non_snake_case)]
```

Comment:
I think, for the sake of consistency we shouldn't use it

Reply:
done

Reply:
The names must match the fields in the JSON responses from DDC.

Reply:
[done](https://github.com/Cerebellum-Network/pos-network-node/pull/66/files#diff-aff5dab0057e0c5f4dfde144e3cfd41744384afd00ea909ba97ce6058ba7af39L3-L6)